Gemfile.lock

```diff
@@ -191,6 +191,8 @@ GEM
       racc (~> 1.4)
     nokogiri (1.14.1-arm64-darwin)
```

Comment:
This is odd, should be locked to the linux version...

Reply:
Ok, considering you're already there, can you remove Gemfile.lock and rebuild the image and see if that works?

Reply:
done